README.md

```diff
@@ -247,120 +232,197 @@ Merging capability scopes MUST follow set semantics, where the result includes a
+| Field | Type                         | Description                                 | Required |
+|-------|------------------------------|---------------------------------------------|----------|
+| `ucv` | `String`                     | UCAN Semantic Version (`0.2.0`)             | Yes      |
+| `iss` | `String`                     | Issuer DID (sender)                         | Yes      |
+| `aud` | `String`                     | Audience DID (receiver)                     | Yes      |
+| `nbf` | `Integer`                    | Not Before UTC Unix Timestamp (valid from)  | No       |
+| `exp` | `Integer \| null`            | Expiration UTC Unix Timestamp (valid until) | Yes      |
+| `nnc` | `String`                     | Nonce                                       | No       |
+| `fct` | `{String: Any}`              | Facts (asserted, signed data)               | No       |
+| `cap` | `{URI: {Ability: [Object]}}` | Capabilities                                | Yes      |
+| `prf` | `[CID]`                      | Proof of delegation (hash-linked UCANs)     | No       |
+
+
+### 3.2.1 Version
+
+The `ucv` field sets the version of the UCAN specification used in the payload.
 
-### 3.2.1 Principals
+### 3.2.2 Principals
 
-The `iss` and `aud` fields describe the token's principals. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with the private key associated with the DID in the `iss` field. Implementations MUST include the [`did:key` method](https://w3c-ccg.github.io/did-method-key/), and MAY be augmented with [additional DID methods](https://www.w3.org/TR/did-core/).
+The `iss` and `aud` fields describe the token's principals. These can be conceptualized as the sender and receiver of a postal letter. The token MUST be signed with the private key associated with the DID in the `iss` field. Implementations MUST include the [`did:key`] method, and MAY be augmented with [additional DID methods][DID].
 
-If an issuer's DID has more than one key (e.g. [`did:ion`](https://github.com/decentralized-identity/ion), [`did:3`](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-79/CIP-79.md)), the key used to sign the UCAN MUST be made explicit, using the [DID fragment](https://www.w3.org/TR/did-core/#fragment) (the hash index).
+The `iss` and `aud` fields MUST contain a single principal each.
 
-The underlying key types RSA, ECDSA, and EdDSA MUST be supported. Use of ECDSA is supported but [RECOMMENDED that another key type be preferred](https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Security).
+If an issuer's DID has more than one key (e.g. [`did:ion`], [`did:3`]), the key used to sign the UCAN MUST be made explicit, using the [DID fragment] (the hash index).
+
+The underlying key types RSA, ECDSA, and EdDSA MUST be supported. Use of ECDSA is supported but [RECOMMENDED that another key type be preferred][ECDSA security].
```

Comment:
Does RSA _have_ to be supported? We don't support RSA keys generally for instance 🤔 

(we actually don't support EdDSA yet either but will soon 😅)